styles/globals.css

```diff
@@ -1,16 +1,36 @@
-html,
-body {
-  padding: 0;
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,body, body * { 
+  font-family: 'Open Sans', sans-serif; 
+  color:inherit;
+  background-color:inherit;
+ font-size: 62.5%;
+}
+
+li {
+  list-style: none;
 }
 
 a {
-  color: inherit;
   text-decoration: none;
 }
+button {
+background: transparent;
+border: none;
+cursor: pointer;
+padding: 0;
+}
+
+button:hover,
+button:focus {
+  background:transparent;
+}
+
+button:focus {
+  outline: none;
 
-* {
-  box-sizing: border-box;
```

Comment:
I would delete everything over here except for lines 1-13, and update them with the style guide on the Figma